lib/Sema/TypeCheckStorage.cpp

```diff
@@ -676,7 +676,7 @@ static Expr *buildStorageReference(AccessorDecl *accessor,
     auto *backing = var->getPropertyWrapperBackingProperty();
 
     // Error recovery.
-    if (!backing)
+    if (!backing || backing->getType()->hasError())
```

Comment:
You can do `backing->isInvalid()` here (which does this internally).

Reply:
Ah, thank you!